applications/matter_weather_station/sample.yaml

```diff
@@ -2,28 +2,35 @@ sample:
   name: Matter Weather Station
   description: Matter Weather Station application
 tests:
-  # Excluded in quarantine.yaml to limit resources usage in integration builds
-  applications.matter_weather_station.debug:
+  # Excluded in quarantine.yaml to limit resource usage in integration builds
+  applications.matter_weather_station.debug.no_fd:
```

Comment:
Can you add comment that this configuration need to be used for testing Thingy e.g. DFU between different Thingy apps?